build.sh

```diff
@@ -5,41 +5,40 @@ set -euo pipefail
-  echo -e "\tz790p_ddr5     - build Dasharo image compatible with MSI PRO Z790-P (WIFI)"
-  echo -e "\tvp66xx         - build Dasharo for Protectli VP66xx"
-  echo -e "\tvp46xx         - build Dasharo for Protectli VP46xx"
-  echo -e "\tvp32xx         - build Dasharo for Protectli VP32xx"
-  echo -e "\tvp2430         - build Dasharo for Protectli VP2430"
-  echo -e "\tvp2420         - build Dasharo for Protectli VP2420"
-  echo -e "\tvp2410         - build Dasharo for Protectli VP2410"
-  echo -e "\tV1210          - build Dasharo for Protectli V1210"
-  echo -e "\tV1410          - build Dasharo for Protectli V1410"
-  echo -e "\tV1610          - build Dasharo for Protectli V1610"
-  echo -e "\tapu2           - build Dasharo for PC Engines APU2"
-  echo -e "\tapu3           - build Dasharo for PC Engines APU3"
-  echo -e "\tapu4           - build Dasharo for PC Engines APU4"
-  echo -e "\tapu6           - build Dasharo for PC Engines APU6"
-  echo -e "\toptiplex_9010  - build Dasharo compatible with Dell OptiPlex 7010/9010"
-  echo -e "\tqemu           - build Dasharo for QEMU Q35"
-  echo -e "\tqemu_full      - build Dasharo for QEMU Q35 with all menus available"
+  echo -e "\tz690a_ddr4             - build Dasharo image compatible with MSI PRO Z690-A (WIFI) DDR4"
+  echo -e "\tz690a_ddr5             - build Dasharo image compatible with MSI PRO Z690-A (WIFI)"
+  echo -e "\tz790p_ddr4             - build Dasharo image compatible with MSI PRO Z790-P (WIFI) DDR4"
+  echo -e "\tz790p_ddr5             - build Dasharo image compatible with MSI PRO Z790-P (WIFI)"
+  echo -e "\tvp66xx                 - build Dasharo for Protectli VP66xx"
+  echo -e "\tvp46xx                 - build Dasharo for Protectli VP46xx"
+  echo -e "\tvp2420                 - build Dasharo for Protectli VP2420"
+  echo -e "\tvp2410                 - build Dasharo for Protectli VP2410"
+  echo -e "\tV1210                  - build Dasharo for Protectli V1210"
+  echo -e "\tV1410                  - build Dasharo for Protectli V1410"
+  echo -e "\tV1610                  - build Dasharo for Protectli V1610"
+  echo -e "\tapu2                   - build Dasharo for PC Engines APU2"
+  echo -e "\tapu3                   - build Dasharo for PC Engines APU3"
+  echo -e "\tapu4                   - build Dasharo for PC Engines APU4"
+  echo -e "\tapu6                   - build Dasharo for PC Engines APU6"
+  echo -e "\toptiplex_9010_uefi     - build Dasharo compatible with Dell OptiPlex 7010/9010 (UEFI)"
+  echo -e "\toptiplex_9010_seabios  - build Dasharo compatible with Dell OptiPlex 7010/9010 (SeaBIOS)"
+  echo -e "\tqemu                   - build Dasharo for QEMU Q35"
+  echo -e "\tqemu_full              - build Dasharo for QEMU Q35 with all menus available"
 }
 
 SDKVER="2023-11-24_2731fa619b"
 
 
 function build_optiplex_9010 {
-  DEFCONFIG="configs/config.dell_optiplex_9010_sff_uefi_txt"
-  FW_VERSION=v0.1.0-rc1
+  local CONFIG=$1
```

Comment:
Lets keep it named `DEFCONFIG` to mark that it isn't full config.

Reply:
[Fixed](https://github.com/Dasharo/coreboot/pull/583/commits/a9f663219c418f947db65114e0dd7c895ec98696)